src/query_ctrl.ts

```diff
@@ -49,6 +51,27 @@ export class MetaQueriesQueryCtrl extends QueryCtrl {
 
     this.queryTypes = _.keys(this.queryTypeValidators);
 
+    if (!this.target.orderType) {
+      this.target.orderType = 'Top';
+    }
+    
+    this.orderTypes = ['Top', 'Bottom'];
+ 
+    if (!this.target.orderSize && this.target.orderSize !== 0) {
+      this.target.orderSize = 5;
+    }
+ 
+    this.orderSizes = [
```

Comment:
why are we limiting the results by orderSize ? can you help me understand the use case for it ?

Reply:
I use this so some insignificant results don't clutter a graph. 
For example, in the graph on this PR description I want to see the hit rate on some redis servers. But the only servers that are interesting to me are the ones with low hit rate (low hit rate is an issue for me).
The orderSize helps me unclutter the graph and visualize faster the servers where I need to take action.

Reply:
👍